.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "10"
```

Comment:
Do we need to check this against more than that? Do we care?

Reply:
Nah, this is a node only module anyways.